app.js

```diff
@@ -1,26 +1,20 @@
+const userRoutes = require('./routes/user.routes')
 const express = require('express')
-const registrationRoutes = require('./routes/registration.routes')
+var cors = require('cors')
+const mongodbConnect = require('./mongoDB/mongodb.connect')
 const app = express()
-const mongoDB = require('./config/db')
 
-mongoDB.connect()
-var cors = require('cors')
+mongodbConnect.connect()
 app.use(cors())
 app.options('*', cors())
-
 app.use(express.json({ extended: false }))
-
-app.use('/api/accounts', registrationRoutes)
+app.use('/api/v1', userRoutes)
```

Comment:
I would change the endpoint to be '/api/v1/user' so we can determine what this endpoint is associated with. Later on we can create an endpoint that works specifically with restaurants like '/api/v1/restaurants' and in this endpoint we can add, edit or remove restaurants. We can do the same for menu items. These 3 things are the major backbones of our website.